index.html

```diff
@@ -1235,53 +1235,36 @@
+aligns with the user's wishes and interests.</span>
+</div>
 
 On the Web, user agents and sites often communicate data outside what is needed to load and display a
-page. These <dfn data-lt="ancillary use">ancillary uses</dfn> include:
-
-* browser telemetry, including:
-  * performance measurements,
-  * measuring feature usage,
-  * debugging site, browser and networking issues, and
-  * detecting security problems or attacks;
-* site telemetry, including:
-  * performance measurements,
-  * analytics about usage and visitors,
-  * debugging site issues, and
-  * detecting security problems or attacks;
-* advertising and social media sharing, including:
-  * measurement,
-  * targeting,
-  * operating ad auctions,
-  * personalization, and
-  * fraud prevention;
-* performance, including caching and prefetching content; or
-* software updates.
-
-<dfn>Telemetry and analytics</dfn>, including browser telemetry and site telemetry, are a common
-cluster of ancillary uses.
-
-In some cases, mechanisms for sharing data are provided in order to minimize some other data
-collection mechanism identified as more intrusive or costly.
-
-Different users will want to share different kinds and amounts of data with websites. For some uses,
-some people would reasonably anticipate sharing such data or identify that use as a
-commonly-accepted contextual norm and some people would be willing to participate (for example,
-might have explicitly decided to contribute if they had the time and understanding to consider
-details about such a use) and some people would identify those uses as supporting their own goals.
-
-Aggregated or [=de-identified=] data is often more likely to be
-acceptable in these senses and often still contributes to the collective benefit while minimizing
-privacy threats to a particular individual (see <a href="#principle-collective-privacy">collective
-privacy</a>). But some people might object anyway, in particular instances or in general.
+page. These include:
+
+<dfn>Telemetry and analytics</dfn>, including browser telemetry and site telemetry, are common
+categories of <dfn data-lt="ancillary use">ancillary uses</dfn> of data.
+
+In some cases, aggregate data collection mechanisms
+can be used in place of other
+more intrusive or costly data collection mechanisms.
+
+Different users will want to share different kinds and amounts of data with websites.
+In some case, users might reasonably anticipate of some data being collected
+for an [=ancillary uses=] because of a contextual norm the user approves of. In such cases,
+users are identifying such ancillary uses as aligned with their own goals.
+
+Aggregated or [=de-identified=] data may contribute to a collective benefit in a way
+that reduces privacy threats to individuals (see <a href="#principle-collective-privacy">collective
+privacy</a>). This may make users more interested in contributing their data to aggregates.
 
```

Comment:
Suggested addition from @dmarti's #215.

```suggestion

A [=user agent=] must act in the interest of its user when reporting aggregated data and should
not use the fact that data is aggregated as an excuse to share more data than is in that user's interest.

```

I haven't double-checked that this is the best place to insert this in the new flow.

Reply:
Does this add anything to our definition of the duty of loyalty? I'm not sure we should add "don't violate the duty of loyalty by making up excuses" to each section.

Reply:
interested -> comfortable